sql/core/src/test/scala/org/apache/spark/sql/execution/UnsafeRowSerializerSuite.scala

```diff
@@ -125,7 +130,7 @@ class UnsafeRowSerializerSuite extends SparkFunSuite with LocalSparkSession {
     assert(sorter.numSpills > 0)
 
     // Merging spilled files should not throw assertion error
-    sorter.writePartitionedFile(ShuffleBlockId(0, 0, 0), outputFile)
+    sorter.writePartitionedMapOutput(0, 0, mapOutputWriter)
```

Comment:
I remember we remove local directories when the JVM (for testing) is shut down. But would be great to confirm that before we merge this.

Reply:
Oh, just read https://github.com/apache/spark/pull/39368?notification_referrer_id=NT_kwDOAGLXhbI1MjMzNjcxNTQ0OjY0Nzc3MDE&notifications_query=is%3Aunread#issuecomment-1373501036. LGTM too